### DISCUSSION TITLE
Combine Fill level when Unloader is called

### DISCUSSION THREAD

**Discussion by Roomzie** [Courseplay/Courseplay_FS22#3248] [General]
Hi there! 

I´m using the CP unloading of the Combine because in my opinion you guys did a way greater job at setting up the ai so it doesnt stop every 100m to check where the Combine is at the moment. 

However i cant seem to find an option where i can setup the fill level at which the Unloader gets called, i tried to set it up in the autodrive settings but it still calls the Unloader at 70% (i´d like to call it as soon as it´s possible so the unloader always drives beside the combine) 

Thanks in Advance for your help.

**Comment by coatsy35** [comment]
it's on the CP settings page when you have a combine selected, near the setting that allows you to turn on unloading on the first headland etc...

**Reply by Roomzie** [reply]
![20240519121848_1](https://github.com/Courseplay/Courseplay_FS22/assets/95157624/03c4f61b-67ec-414b-8fb4-c0b266b31747)
I dont see that option, thats my problem

**Comment by Tensuko** [comment]
That is a fixed value and cant be changed.
It is on purpose.
Driving next to combine all the time is not intended as well.

**Reply by Roomzie** [reply]
yeah i get that, that wasnt really 100% what i meant but the combine is set to call the unloader at 70%, now if the combine has a small tank that means he will in 9/10 cases have to stop since he is full before the unloader arrives, i just thought putting it on like 40% would make more sense in terms of efficiency but i´m sure you have your reasons why that value is set to 70. Thanks alot and you guys are doing great work <3
